src/Popover/helpers/calculatePopoverPosition.js.flow

```diff
@@ -0,0 +1,13 @@
+// @flow
+
+import type { PositionsCore, AnchorsCore } from "../index.js.flow";
+
+export type Positions = {|
+  positions: PositionsCore,
+|};
+
+export type Anchors = {|
+  anchors: AnchorsCore,
+|};
+
+declare export default ({}, {}, string) => [PositionsCore, AnchorsCore];
```

Comment:
Preferably, declare and export type and use it for declaring the default export:
```
export type MyFunctionName = (something: boolean) => [PositionsCore, AnchorsCode];

declare export MyFunctionName;
```
You can also then import this type and use it:
```
import type { MyFunctionName } from "."
const myFunctionName: MyFunctionName = (something: boolean) => ["something", "something"];
```